### PR TITLE
Seed is an integer

### PR DESCRIPTION
**Pull request by jmeyers314**
The seed for the rng that introduces errors in the m1m3 lookup table (within batoid_rubin) needs to be an integer.  I didn't notice this before as the test case I created before elided that RNG entirely since the error was set to 0.0.

The fix here is a bit of a hack; I should probably write a more extensive config parser for this someday.  But this should work for the moment and lets me hand off the current main branch of ImSim to the AOS team for further integration.

### CODE REVIEW

**Review by rmjarvis — APPROVED** [review]
I'm approving, but I laid out what I think is probably a better design pattern, if you want to try to clean it up now rather than wait for later.

File: imsim/telescope_loader.py [hunk 1]
```diff
@@ -145,6 +145,8 @@ def parse_fea(config, base, telescope):
                 v[kk], safe = ParseValue(v, kk, base, Angle)
             elif kk == 'dof':
                 v[kk], safe = ParseValue(v, kk, base, None)
+            elif kk == 'seed':
+                v[kk], safe = ParseValue(v, kk, base, int)
```
[the remark this answers is not in the document]
Comment (jmeyers314) [reply imsim/telescope_loader.py]: Thanks.  That's actually less work than I figured.

I went ahead and attached the `req` and `opt` lists directly as method attrs in batoid_rubin.  New release should pop up soon.

[the remark this answers is not in the document]
Comment (rmjarvis) [reply imsim/telescope_loader.py]: Once the new batoid version is ready, I guess restart the CI action, since (of course) it's failing now.

**Comment by jmeyers314** [comment]
Looks like it's failing now because `list` isn't an acceptable type for GetAllParams?

(And as an aside, I had already tried to fix this by setting the type to None, but messed up the tagging and release on github so my first attempt with type list was uploaded to Pypi and grabbed by conda).

**Comment by rmjarvis** [comment]
I see.  Yeah, it would work if `config['dof']` is an actual list.  But not if it needs to be constructed (via an Eval here).  I'll make an issue in GalSim to fix this.  (Using None instead should be a valid workaround for now.)

**Pull request merged**